.github/workflows/clean-aws-resources.yml

```
@@ -288,3 +288,21 @@ jobs:
       - name: Clean old Log Groups
         working-directory: tool/clean
         run: go run ./clean_log_group/clean_log_group.go 
+  clean-security-groups:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: ${{ secrets.TERRAFORM_AWS_ASSUME_ROLE }}
+          aws-region: us-west-2
+
+      - name: Clean Old Security Groups
+        working-directory: tool/clean
+        run: go run ./clean_security_group/clean_security_group.go
```

Can we add some error handling here
```suggestion
        run: |
          set -e
          go run ./clean_security_group/clean_security_group.go || { echo "Failed to clean security groups"; exit 1; }
```